conf/modules/call_snv_deepvariant.config

```diff
@@ -37,20 +37,14 @@ process {
     withName: '.*CALL_SNV_DEEPVARIANT:REMOVE_DUPLICATES_GL' {
         ext.args = '--output-type z --rm-dup none'
         ext.prefix = { "${meta.id}_split_rmdup" }
```

Comment:
Sure you don't want `REMOVE_DUPLICATES_GL` to publish?

Reply:
Not really. We only publish the SNV calls from deepvariant after excluding mitochondrial variants. Publishing the output from REMOVE_DUPLICATES_GL along with other variant files can cause confusion. 😄